init.sls

```diff
@@ -48,6 +50,16 @@ validate-nginx-config:
     - require:
       - pkg: nginx
 
+# Ensure cache directory exists
```

Comment:
Can you elaborate the comment to say that this is so it's there for the systemd hardened service? Otherwise this might be confusing in the future.